src/integrators/sdirk_integrators.jl

```diff
@@ -390,11 +407,8 @@ end
   end
 
   @. z = u - uprev
-  iter = 0
-  κ = cache.κ
-  tol = cache.tol
-
-  iter += 1
+  iter = 1
+  @. u = uprev + z
```

Comment:
Yeah it was a mistake. These implementations haven't been cleaned up yet so there's some details that I've opened issues for to investigate. This is one of those things I noticed that can probably be removed, but didn't because I'd want to regression test when doing so.